content/blog/announcing-aqr-as-sponsor-2018.md

```diff
@@ -1,7 +1,7 @@
 Title: Announcing AQR as Platinum Sponsor
 Date: 2018-07-14 10:00
 Category: 2018
-Slug: announcing-aqr-as-platinum-sponsor
+Slug: announcing-aqr-as-platinum-sponsor-2018
```

Comment:
Was this causing any issue ?

Reply:
Yeah, there's another blog post in 2019 with the same slug that was causing a warning.